hazelcast/src/test/java/com/hazelcast/dataconnection/HazelcastDataConnectionTest.java

```diff
@@ -261,4 +260,8 @@ private static DataConnectionConfig sharedDataConnectionConfigFromFile(String cl
             throw new RuntimeException(e);
         }
     }
+
+    private static String readFile() throws IOException {
```

Comment:
I like this, but could you also move `.replace("$CLUSTER_NAME$", clusterName)` in as well?

Reply:
Let's leave what will needs to be replaced to individual test logic.  The readFile() function is short and clear as it is